ls1intum/Artemis
# `Programming exercises`: Update Java template to JDK17

Update Java programming exercises to JDK17 with a new docker image

### Checklist
#### General

#### Changes affecting Programming Exercises

### Steps for Testing

Prerequisites:
- 1 Instructor
- 2 Students

1. Create a programming exercise with Java (one with and one without static code analysis)
2. Check if the creation works correctly
3. Participate with different students and check if everything works correctly 

### Review Progress





#### Code Review
#### Manual Tests
